src/cluster.py

```diff
@@ -173,6 +178,15 @@ def _create_directory(self, path: str, mode: int) -> None:
         os.chmod(path, mode)
         self._change_owner(path)
 
+    def _create_user_home_directory(self) -> None:
+        """Creates the user home directory for the snap_daemon user.
+
+        This is needed due to https://bugs.launchpad.net/snapd/+bug/2011581.
+        """
+        subprocess.run("mkdir -p /home/snap_daemon".split())
+        subprocess.run("chown snap_daemon:snap_daemon /home/snap_daemon".split())
+        subprocess.run("usermod -d /home/snap_daemon snap_daemon".split())
```

Comment:
pgBackRest needs to run as `snap_daemon`, otherwise it breaks PortgreSQL permissions (blocking it startup after a restart).